EDU-16074: Delivery Promise (2.1 and 2.2) (#2179)
vtexdocs/dev-portal-content: docs/faststore/components/features/delivery-promise/implementation.mdx

  },
```

* `enabled`: When set to **true**, Delivery Promise activates, allowing the store to provide delivery estimates based on the user's location.

> * `mandatory`: When set to **false**, providing a postal code is optional for the user. Users can browse and add items to the cart without entering their postal code, although the accuracy of delivery information may be limited.

Reviewer: how about explaining what happens when set to **true**, and then explain the default? Something like:

```suggestion
* `mandatory`: When set to **true**, users must provide a postal code. By default, this is **false**, allowing users to browse and add items to the cart without entering their postal code. However, without a postal code the accuracy of delivery information may be limited.
```

        geoCoordinates: null,
        person: null,
      },
      ```
## Managing Delivery Promise filters and labels

Delivery Promise [filters](https://developers.vtex.com/docs/guides/faststore/delivery-promise-components) help shoppers narrow products by delivery/pickup availability for their location, while [badges](https://developers.vtex.com/docs/guides/faststore/delivery-promise-components) visually indicate shipping/pickup availability for the shopper's location when Delivery Promise data exists.

By default, filters and badges are active once Delivery Promise is enabled and implemented in the store. However, you can deactivate them (Global Filter by Pickup Point, Delivery Option, and Badges) and edit their labels in the Headless CMS. To do this, follow these steps.

>ℹ️ For more details on filters and badges, see the [Working with Delivery Promise components](https://developers.vtex.com/docs/guides/faststore/delivery-promise-components) guide.

1. Access the VTEX Admin and go to **Storefront > Headless CMS**.

> 2. Click on the **Global Sections** Content Type.

Reviewer 2: ```suggestion
2. Click the **Global Sections** Content Type.
```

        person: null,
      },
      ```
## Managing Delivery Promise labels

Delivery Promise [filters](https://developers.vtex.com/docs/guides/faststore/delivery-promise-components) help shoppers narrow products by delivery/pickup availability for their location, while [badges](https://developers.vtex.com/docs/guides/faststore/delivery-promise-components) visually indicate shipping/pickup availability for the shopper's location when Delivery Promise data exists.

> By default, filters and badges are already active once Delivery Promise is implemented in the store. However, you can edit their labels in the Headless CMS. To do this, follow these steps.

Reviewer 3: ```suggestion
By default, filters and badges are already active once Delivery Promise is implemented in the store. However, you can deactivate them (Global filter by pickup point, Delivery Option, and Badges) and edit their labels in the Headless CMS. To do this, follow these steps.
```

>ℹ️ For more details on filters and badges, see the [Working with Delivery Promise components](https://developers.vtex.com/docs/guides/faststore/delivery-promise-components) guide.

1. Access the VTEX Admin and go to **Storefront > Headless CMS**.
2. Click on the **Global Sections** Content Type.
3. Click the **Settings** tab.

> 4. Go to the **Delivery Promise** section and change the labels to suit your scenario.

Reviewer 3: ```suggestion
4. Go to the **Delivery Promise** section and change the labels and settings to suit your scenario.
```
